src/app.js

```diff
@@ -1,5 +1,218 @@
 const express = require('express');
+const strings = require('./lib/strings.js');
+const numbers = require('./lib/numbers.js');
+const bool = require('./lib/booleans.js');
+const arr = require('./lib/arrays.js');
 
 const app = express();
 
+app.use(express.json()); // FOR POST
+// app.use(bodyParser.json());
+
+app.get('/strings/hello/:basename', (req, res) => {
+  const word = strings.sayHello(req.params.basename);
+
+  res.json({ result: `${word}` }).sendStatus(200);
+});
+
+app.get('/strings/upper/:basename', (req, res) => {
+  const word = strings.uppercase(req.params.basename);
+
+  res.json({ result: `${word}` }).sendStatus(200);
+});
+
+app.get('/strings/lower/:basename', (req, res) => {
+  const word = strings.lowercase(req.params.basename);
+
+  res.json({ result: `${word}` }).sendStatus(200);
+});
+
+app.get('/strings/first-characters/:basename', (req, res) => {
+  const length = req.query.length ? parseInt(req.query.length) : 1;
+  const word = strings.firstCharacters(req.params.basename, length);
+  res.json({ result: `${word}` }).sendStatus(200);
+
+  /* if (req.query.length) {
+    const n = parseInt(Object.values(req.query).join());
+    const word = strings.firstCharacters(req.params.basename, n);
+    res.json({ result: `${word}` }).sendStatus(200);
+  } else {
+    const word = strings.firstCharacter(req.params.basename);
+    res.json({ result: `${word}` }).sendStatus(200);
+  } */
+});
+//-----------------------------------------------
+app.get('/numbers/add/:param1/and/:param2', (req, res) => {
+  const num1 = parseInt(req.params.param1);
+  const num2 = parseInt(req.params.param2);
```

Comment:
Number() can also be used as an alternative to parseInt(). Do you know the difference between them?

Reply:
Not really, sorry. Will defo look it up. from what little I understand Number would be an object wrapper so it did not occur to me to use it. In fact even later during the project I kept doing .isNaN() which wouldn't work until I figured much later after much toiling that it should be Number.isNaN()